src/unit/engine/UberFilesizeTestEngine.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+* Checks if any of modified/added files are larger than the given limit.
+* Filesize limit can be specified as a runtime config in ".arcconfig" as
+* following:
+* { ...
+*   "unit.engine": "UberMultiTestEngine",
+*   "unit.engine.multi.engines":
+*   [
+*     ...,
+*     {
+*       "engine": "UberFilesizeTestEngine",
+*       "MAX_FILESIZE_LIMIT" : 1000000
+*     },
+*     ...
+*   ],
+*   ...
+* }
+*/
+
+final class UberFilesizeTestEngine extends ArcanistUnitTestEngine {
+
+    const MAX_FILESIZE_LIMIT_KEY = 'MAX_FILESIZE_LIMIT';
```

Comment:
arc seems to use lower case dot syntax, so maybe change this to `max_filesize_limit`